portal/src/graphql/portal/PasswordSettings.tsx

```diff
@@ -408,6 +429,30 @@ export default function PasswordSettings<T extends State>(
     [setState]
   );
 
+  const onChangeExpiryForceChangeDays = useCallback(
+    (_e, value: string | undefined) => {
+      if (value == null) {
+        return;
+      }
+      setState((s) =>
+        produce(s, (s) => {
+          s.authenticatorPasswordConfig.expiry ??= {};
+          s.authenticatorPasswordConfig.expiry.force_change ??= {};
+          s.authenticatorPasswordConfig.expiry.force_change.duration_since_last_update =
+            tryProduce(
+              s.authenticatorPasswordConfig.expiry.force_change
+                .duration_since_last_update,
+              () => {
+                const num = parseNumber(value);
```

Comment:
It does not allow me to delete all text, and re-enter a new number. When I delete the last digit, the toggle becomes disabled, and the text field is blurred.

I think the behavior should be

1. Only allow digits to be entered in the text field.
2. Allow the text field to be empty.
3. If the text field is blurred and it is empty, then change the toggle to false.
4. If the text field is blurred and it is 0, then change the toggle to false.
5. When the toggle is false, the text field should be empty, not contain a 0.